templates/default/snapraid_runner.conf.erb

```diff
@@ -1,14 +1,17 @@
+<%
+path_separator = node['platform'] == 'windows' ? '\\' : '/'
+%>
 [snapraid]
 ; path to the snapraid executable (e.g. /bin/snapraid)
-executable = /usr/local/bin/snapraid
+executable = <%=node['snapraid']['install_directory'] + path_separator%>snapraid<%= node['platform'] == 'windows' ? '.exe' : ''%>
```

Comment:
Also good shout, much prettier.
Though how do you mean testable in chef-spec? It'll get tested by the with_content regex either way won't it? (or at least it will now I've added it!)